src/components/IndividualPageContent/ContentRow.tsx

```diff
@@ -20,29 +21,35 @@ export default function ContentRow({
     <Box>
       <Grid
         container
-        rowSpacing={1}
+        rowSpacing={0.5}
         columnSpacing={4}
         alignItems="center"
         key={i}
       >
-        <Grid item xs={12} md={3}>
-          <Stack direction="row">
-            {tooltip}
+        <Grid xs={12} sm={3}>
+          <Box>
             <Typography variant="body2" color={grey[450]}>
               {title}
+              <Box
+                component="span"
+                sx={{display: "inline", whiteSpace: "nowrap"}}
+              >
+                &nbsp;
+                <Box sx={{display: "inline-block"}}>{tooltip}</Box>
```

Comment:
Looks great! Thanks! One last thought: would it be even better to make the info icon center-aligned with the label?

Reply:
@moonclavedev please feel free to land it. I can come back to this later. I'd love to have the changes landed for the dogfooding session/instruction I'll soon be sending out.